apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/ManagedTracerFactory.cs

```diff
@@ -52,22 +52,22 @@ TraceIdFactory traceIdFactory
         }
 
         /// <inheritdoc />
-        public IManagedTracer CreateTracer(TraceHeaderContext headerContext)
+        public IManagedTracer CreateTracer(ITraceContext traceContext)
```

Comment:
This is a breaking change - we'll need to add an overload instead.

Reply:
Thanks - had completely missed that. (Caught it another time where I'd started adding a similar comment...)